modules/store/src/selector.ts

```diff
@@ -601,8 +603,19 @@ export function createFeatureSelector<T, V>(
 export function createFeatureSelector(
   featureName: any
 ): MemoizedSelector<any, any> {
-  return createSelector(
-    (state: any) => state[featureName],
-    (featureState: any) => featureState
-  );
+  return createSelector((state: any) => {
+    const featureState = state[featureName];
+    if (isDevMode() && featureState === undefined) {
+      console.error(
```

Comment:
At other places we prefix the message to mention it's coming from NgRx (and the package name), and we usually use `console.warn`.

https://github.com/ngrx/platform/blob/8f05f1f0c24bdf84b65d18e243ccf62caf480d20/modules/entity/src/utils.ts#L8

Reply:
Changed to warning in 068ed28c14b03fe85bcdaac07610d20bbf77b46c.